doc/ref_impl/cntt-ri2/chapters/chapter04.md

```diff
@@ -54,7 +54,36 @@ After the hosts have been provisioned successfully, one can set up host networki
 
 <a name="4.3.2"></a>
 ### 4.3.2 Kubernetes Provisioning
-<Intel BMRA/CNF Testbed>
+
+For provisioning Kubernetes in the Reference Implementation, existing tools should be used to set up a baseline cluster. The initial goal is to cover at least a subset of the requirements described in the [Reference Architecture 2 (RA-2)](../../../ref_arch/kubernetes), with the possibility of extending the implementation to cover more requirements over time.
```

Comment:
shouldn't the RI meet all of the RA requirements or at least all of the MUST?

Reply:
or maybe just state that as the end goal?

Reply:
If I recall correctly this was briefly brought up during the meetings last week. I agree the end goal is to cover all MUSTs, but it adds a lot of requirements (in particular on the infrastructure) that requires lab setups resembling those of production systems.
I feel we need to take more of a step-wise approach, but to add more functionality and cover more requirements, but also to feed more details back into the RI2.
I will add a bit more details to this paragraph to make the direction more clear

Reply:
I've updated the text to make it more clear that the goal is to cover RA2